### ISSUE TITLE
exporter/datadog: doesn't support metrics and does not log this fact

### ISSUE THREAD

**Issue by flands** [open-telemetry/opentelemetry-service#124]
For more details see: https://github.com/census-instrumentation/opencensus-service/issues/454

**Comment by MohamedElqdusy** [comment]
As stated [here](https://github.com/open-telemetry/opentelemetry-service/blob/master/docs/migrating-from-opencensus.md#migration-tasks) Datadog is not one of the core exporters, isn't it?

**Comment by flands** [comment]
That is correct -- it will move to the contrib repo once created. Temporary move over to clean up OC issues.
